### ISSUE TITLE
failed to import plugin code: context deadline exceeded

### ISSUE THREAD

**Issue by WLun001** [traefik/traefik#8199]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?
- Try to write a middleware plugin that import `golang.org/x/text`

### Error
[Hit error on test](https://github.com/WLun001/i18n-rewrite/runs/2864353586) 
```
yaegi test -v .

scope.go:209: nil type
panic: nil type [recovered]
	panic: /home/runner/work/i18n-rewrite/i18n-rewrite/go/src/github.com/WLun001/i18n-rewrite/vendor/golang.org/x/text/internal/tag/tag.go:55:8: CFG post-order panic: nil type

```
and [hit another error](https://github.com/WLun001/i18n-rewrite/issues/2), `plugin: failed to import plugin code: context deadline exceeded` 

This is the output running on [docker-compose locally](https://github.com/WLun001/i18n-rewrite/blob/ed575f4cc22fefc7df1db7b8dff37ab16b7fd318/docker-compose.dev.yaml#L7-L27)

```
2021/06/19 14:25:55 traefik.go:76: command traefik error: github.com/WLun001/i18nrewrite: failed to import plugin code "github.com/WLun001/i18nrewrite": 1:21: import "github.com/WLun001/i18nrewrite" error: /plugins/go/src/github.com/WLun001/i18nrewrite/i18n.go:11:2: import "golang.org/x/text/language" error: /plugins/go/src/github.com/WLun001/i18nrewrite/vendor/golang.org/x/text/language/language.go:198:3: undefined selector: RemakeString
```

Could be related to https://github.com/traefik/yaegi/issues/1158

**Comment by WLun001** [comment]
This is the output running on docker-compose locally

https://github.com/WLun001/i18n-rewrite/blob/ed575f4cc22fefc7df1db7b8dff37ab16b7fd318/docker-compose.dev.yaml#L7-L22


```
2021/06/19 14:25:55 traefik.go:76: command traefik error: github.com/WLun001/i18nrewrite: failed to import plugin code "github.com/WLun001/i18nrewrite": 1:21: import "github.com/WLun001/i18nrewrite" error: /plugins/go/src/github.com/WLun001/i18nrewrite/i18n.go:11:2: import "golang.org/x/text/language" error: /plugins/go/src/github.com/WLun001/i18nrewrite/vendor/golang.org/x/text/language/language.go:198:3: undefined selector: RemakeString
```

**Comment by ldez** [comment]
Hello,

the error `context deadline exceeded` comes because your plugin takes more than 5s to load.

But in all cases, your plugin has a problem related to the use of `golang.org/x/text` inside yaegi.

Then the issue is on the Yaegi side.

Closed in favor of traefik/yaegi#1158
